index.html

```diff
@@ -31,9 +31,133 @@ <h2 class="intro">Hey there. I'm Selma <br><span>I am a software<br> developer</
+            <li>Ruby on rails </li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+        </ul>
+        <button class="work-button" type="button">See Project            
+        </button>
+    </div>
+    <div class="work-div">
+        <i class="projectimg"></i>
+        <h4>Multi-Post Stories<br>Gain + Glory
+        </h4>
+        <ul class="work-list">
+            <li>Ruby on rails </li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+        </ul>
+        <button class="work-button" type="button">See Project            
+        </button>
+    </div>
+    <div class="work-div">
+        <i class="projectimg"></i>
+        <h4>Multi-Post Stories<br>Gain + Glory
+        </h4>
+        <ul class="work-list">
+            <li>Ruby on rails </li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+        </ul>
+        <button class="work-button" type="button">See Project            
+        </button>
+    </div>
+    <div class="work-div">
+        <i class="projectimg"></i>
+        <h4>Multi-Post Stories<br>Gain + Glory
+        </h4>
+        <ul class="work-list">
+            <li>Ruby on rails </li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+        </ul>
+        <button class="work-button" type="button">See Project            
+        </button>
+    </div>
+    <div class="work-div">
+        <i class="projectimg"></i>
+        <h4>Multi-Post Stories<br>Gain + Glory
+        </h4>
+        <ul class="work-list">
+            <li>Ruby on rails </li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+        </ul>
+        <button class="work-button" type="button">See Project            
+        </button>
+    </div>
+    <div id="about-head">
+        <h2>About me</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, <br>
+        don’t hestiate to contact me.</p>
+        <button class="about-button" type="button">Get My Resume</button>
+    </div>
+    <div class="about-div">
+        <img class="about-icon" src="images/languages.svg" alt="">
+        <h4>Languages</h4>
+        <ul class="about-list">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li> 
+        </ul>
+    </div>
+    <div class="about-div">
+        <img class="about-icon" src="images/frameworks.svg" alt="">
+        <h4>Frameworks</h4>
+        <ul class="about-li">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li> 
+            <li>Selenium</li> 
+        </ul>
+    </div>
+    <div class="about-div">
+        <img class="about-icon" src="images/skills.svg" alt="">
+        <h4>Skills</h4>
```

Comment:
- [ ] In the same light, visit your CSS and give enough margin between the skills icon and the h4 text. See screenshots for clarity. One of the objectives of this project is to be able to reproduce a given design exactly. We shall achieve exactly that.

Your design 
![image](https://user-images.githubusercontent.com/62849872/143690257-ec177d0c-2025-4274-917d-6226db1c3f8c.png)

Figma design 
![image](https://user-images.githubusercontent.com/62849872/143690278-8b1a68e4-e9cc-44d0-89df-a5fca76b06ff.png)